docs/demos/sequential_testing.ipynb

```diff
@@ -0,0 +1,3260 @@
+{
```

Comment:
<p>A quick explanation of why sequential testing is useful (we want to find which features are the most context dependent, and which contexts are the most important), and how we actually achieve this would be good to start with.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>